### ISSUE TITLE
Update to work with TikTokApi 3.3.0

### ISSUE THREAD

**Issue by Coloradohusky** [Coloradohusky/TikUp#13]
```
Traceback (most recent call last):
  File "c:\users\colohusky\appdata\local\programs\python\python38\lib\runpy.py", line 193, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "c:\users\colohusky\appdata\local\programs\python\python38\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "C:\Users\Colohusky\AppData\Local\Programs\Python\Python38\Scripts\tikup.exe\__main__.py", line 7, in <module>
  File "c:\users\colohusky\appdata\local\programs\python\python38\lib\site-packages\tikup\tikup.py", line 142, in main
    tiktoks = downloadUser(username, limit, file)
  File "c:\users\colohusky\appdata\local\programs\python\python38\lib\site-packages\tikup\tikup.py", line 71, in downloadUser
    tiktoks = api.byUsername(username, count=count)
  File "c:\users\colohusky\appdata\local\programs\python\python38\lib\site-packages\TikTokApi\tiktok.py", line 140, in byUsername
    return self.userPosts(data['userId'], data['secUid'], count=count, proxy=proxy, language=language, region=region)
KeyError: 'userId'
```

**Comment by Coloradohusky** [comment]
Once pushed to master, it should work again

**Comment by Coloradohusky** [comment]
Fixed in 2020.06.26
